frontend/catalyst/device/qjit_device.py

```diff
@@ -319,14 +320,40 @@ def __init__(self, original_device):
         for key, value in original_device.__dict__.items():
             self.__setattr__(key, value)
 
-        check_device_wires(original_device.wires)
-
-        super().__init__(wires=original_device.wires)
+        if (original_device.wires is not None) and any(
+            isinstance(wire_label, tuple) and (len(wire_label) >= 2)
+            for wire_label in original_device.wires.labels
+        ):
+            wires_from_cmap = set()
+            for wire_label in original_device.wires.labels:
+                wires_from_cmap.add(wire_label[0])
+                wires_from_cmap.add(wire_label[1])
+            wires_from_cmap = qml.wires.Wires(list(wires_from_cmap))
+            # check_device_wires(wires_from_cmap) not called
+            # since automatic qubit management
+            super().__init__(wires=wires_from_cmap, shots=original_device.shots)
```

Comment:
core PL has deprecated setting shots on devices
```suggestion
            super().__init__(wires=wires_from_cmap)
```